proto/ethermint/evm/v1/evm.proto

```diff
@@ -28,6 +26,34 @@ message Params {
   repeated string active_precompiles = 7;
   // evm_channels is the list of channel identifiers from EVM compatible chains
   repeated string evm_channels = 8 [(gogoproto.customname) = "EVMChannels"];
+  // permissions defines the permission policy of the EVM
+  Permissions permissions = 9 [(gogoproto.customname) = "PermissionsPolicy", (gogoproto.nullable) = false];
```

Comment:
let's call this `AccessControl` which is in line with the [ACL in solidity contracts](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.1/contracts/access/AccessControl.sol)